src/parser/druid/balance/modules/talents/azeritetraits/HighNoon.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import { formatNumber, formatPercentage } from 'common/format';
+import { calculateAzeriteEffects } from 'common/stats';
+import HIT_TYPES from 'game/HIT_TYPES';
+import Analyzer from 'parser/core/Analyzer';
+import AzeritePowerStatistic from 'interface/statistics/AzeritePowerStatistic';
+import SpellLink from 'common/SpellLink';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import StatTracker from 'parser/shared/modules/StatTracker';
+
+const debug = false;
+
+class HighNoon extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    statTracker: StatTracker,
+  };
+  /**
+   * Gives you a buff that increases your solar wrath damage by x for 8 seconds
```

Comment:
should be sunfire, not solar wrath here right?

Reply:
Comments are a little wonky rn since not finished. Will give it a sweeping look over and add more after last 2 traits are done